src/main/maximumLength.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  registerDecorator,
+  ValidationArguments,
+  ValidationOptions,
+  ValidatorConstraint,
+  ValidatorConstraintInterface
+} from 'class-validator'
+
+@ValidatorConstraint()
+export class MaxLengthValidatorConstraint implements ValidatorConstraintInterface {
+  validate (value: any, args?: ValidationArguments): boolean {
+    if (value === undefined || args === undefined) {
```

Comment:
Please remove `args === undefined` and make args non optional as we said on the other PR.